flujo/application/core/step_policies.py

```
@@ -6247,6 +6248,22 @@ async def execute(
             )
             branch_contexts[branch_name] = branch_context
 
+        def _merge_branch_context_into_parent(branch_ctx: Any) -> None:
+            nonlocal context
+            if context is None or branch_ctx is None:
+                return
+            try:
+                from flujo.utils.context import safe_merge_context_updates as _merge
+
+                _merge(context, branch_ctx)
+            except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)